sdk/containerregistry/azure-containerregistry/azure/containerregistry/_exchange_client.py

```diff
@@ -43,13 +43,12 @@ def __init__(self, endpoint, credential, **kwargs):
         if not endpoint.startswith("https://") and not endpoint.startswith("http://"):
             endpoint = "https://" + endpoint
         self._endpoint = endpoint
-        self.credential_scope = kwargs.get("authentication_scope", "https://management.core.windows.net/.default")
+        self.credential_scopes = kwargs.get("credential_scopes", ["https://management.core.windows.net/.default"])
```

Comment:
should this be a private attr?

Reply:
It probably should, but the `ExchangeClient` models are not exposed publicly